pai-management/paiLibrary/imageTool/host-configure.py

```diff
@@ -37,7 +37,7 @@ def load_yaml_config(config_path):
 def read_template(template_path):
 
     with open(template_path, "r") as f:
-        template_data = f.read()
+        template_data = f.read().decode('utf-8')
```

Comment:
>'utf-8' [](start = 40, length = 7)

Is it possible to configure "utf-8" as the default encoding to the whole PAI cluster?

Reply:
The 'str.decode()' method is a good practice to handle unicode in Python. Please see: https://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script. I think the latest change is sufficiently good :)